index.html

```diff
@@ -0,0 +1,147 @@
+                        RO
+                        <br>
+                        <SPan>Music</SPan>
+                    </a>
+                </div>
+                <ul>
+                    <li>
+                        <a href="index.html">Home</a>
+                    </li>
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="#join">Join</a>
+                    </li>
+                    <li>
+                        <a href="#partners">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="">News</a>
+                    </li>
+                    <li>
+                        <a href="">Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+
+    <!-- MAIN SECTION STARTING -->
+    <main>
+        <section class="main">
+            <nav class="Mobile-navbar">
+                <img id="cancelBtn" src="images/Icon - Cancel.svg" alt="">
+                <i id="hamburger" class="fas fa-bars hamburger" aria-hidden="true"></i>
+                <ul id="hamburger-list">
+                    <li>
+                        <a href="index.html">Home</a>
+                    </li>
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="">Join</a>
+                    </li>
+                    <li>
+                        <a href="">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="">News</a>
+                    </li>
+                    <li>
+                        <a href="">Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="container">
+                <div class="content">
+                    <p class="content-hellow">"Hellow! Music World"</p>
+                    <h1 class="main-content-heading">ZIRO MUSIC FESTIVAL NIGHT 2023</h1>
+                    <p class="content-detail">A Ziro Music Festival is a live music performance in front of an audience. 
+                        The performance may be by a single musician, sometimes then called a recital, 
+                        or by a musical ensemble, such as an orchestra, choir, or band. 
+                    </p>
+                    <h2 class="content-secondHeading">2023.10.15(THU) ~ 16(FRI)</h2>
+                    <p class="content-lastline">@ National Music of universe.</p>
+                </div>
+        </div>
+        </section>
+
+        <!-- ------------PROGRAM SECTION STARTING------------- -->
+        <Section class="program" id="join">
+            <h3>Main Program</h3>
+            <div id="allcardS" class="cards"> </div>
+            <button type="button">Join Our Concert</button>
+            <a href="">SEE THE WHOLE PROGRAM</a>
+        </Section>
```

Comment:
The background of the provided design should match the background of your design. The page should look almost identical to the original design, outside of the content differences. Kindly use the same background for this section as in the provided design

Reply:
- [ Optional ] Hello @muneebulrehman As suggested b y the previous reviewer Kindly I recommend you to use the background color just like the Design guidelines So Kindly fix it,
- Your Design background Look likes this : -
![main program 1](https://github.com/anshuman7negi/ConcertPage/assets/93598189/0772bf29-0666-4584-ad31-4b6adfdbbad6)

- The Figma Design Look likes this 
![main program](https://github.com/anshuman7negi/ConcertPage/assets/93598189/f4c70c96-cd10-484a-8f81-e08f8ce803b9)